### DISCUSSION TITLE
Add workspace preview

### DISCUSSION THREAD

**Discussion by mahaveergurjar** [prasanthrangan/hyprdots#479] [Ideas]
Add workspace preview using eww bar
like this 
![screenshot-summer](https://github.com/prasanthrangan/hyprdots/assets/57872034/05b6a788-80ad-47f4-9fe4-eab82c852ef4)

I try but icon are not show that app are running this problem are i am facing
![231111_18h28m35s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/8915341e-b10f-4123-ba1d-09c29ebd1664)

**Comment by T-Crypt** [comment]
@mahaveergurjar 

This is a problem with End-4's use of the icons for this overview. You need the icons from his repository installed in 

/usr/share/icons

I would raise this issue on his repository as @prasanthrangan doesnt manage eww files yet on here.

**Reply by mahaveergurjar** [reply]
OK
